content/docs/2.12/scalers/azure-service-bus.md

```diff
@@ -107,3 +107,14 @@ spec:
     authenticationRef:
         name: azure-servicebus-auth # authenticationRef would need either podIdentity or define a connection parameter
 ```
+
+### Troubleshooting
+
+#### KEDA is unable to get queue information due to "invalid queue runtime properties: no CountDetails element"
+
+When KEDA logs show errors similar to `invalid queue runtime properties: no CountDetails element` it usually is caused because of throttling by Azure Service Bus.
+
+Consider applying one of the following mitigations:
+- Scaling the Azure Service Bus namespace to a higher SKU, or use premium
+- Reduce the polling interval of the ScaledObject/ScaledJob
```

Comment:
This actually means increasing the pollingInterval to reduce the frecuency (ro reduce the amount of calls)